### PR TITLE
Updating cl.yml redirects

### PR DESCRIPTION
**Pull request by shawntanzk**
@anitacaron could you help me make sure these redirects are correctly changed please? :) thanks

### CODE REVIEW

**Comment by shawntanzk** [comment]
hold off merging till we make sure cl-plus can be uploaded to a release

**Review by matentzn — COMMENTED** [review]
I will let @jamesaoverton comment on this but all these "exact" redirects seem to be redundant to the product list to me..

File: config/cl.yml
Comment (jamesaoverton) [inline config/cl.yml:11]: The `products` list should be limited to `IDSPACE.FORMAT`, so `cl-basic.owl` and `cl-basic.obo` should not be allowed. It looks like they've been in here for 7 years, since the initial migration, so I guess we'll keep that. But I don't want to add any more exceptions, so please remove `cl-plus.owl` and `cl-plus.obo`. They should have `exact` entries below.

**Review by jamesaoverton — CHANGES_REQUESTED** [review]
Please remove `cl-plus.*` items from `products`. There's an extra `obophenotype/` in most of the other changes that needs to be rmeoved.

**Comment by jamesaoverton** [comment]
@matentzn No, the `exact` entries are not redundant. The `products` entries control the "top-level" space under <http://purl.obolibrary.org/obo/> and are highly restricted, while the `exact` entries control the "project space" under <http://purl.obolibrary.org/obo/cl/> and are nearly unrestricted.

**Comment by matentzn** [comment]
@jamesaoverton I never knew that! Good to know :) Thanks for the clarification.

**Comment by shawntanzk** [comment]
Think I fixed it, thanks @jamesaoverton & @matentzn

**Review by jamesaoverton — CHANGES_REQUESTED** [review]
I checked some of these updated links and all I got was 404 errors. You should be able to paste the links into a browser and get a file. If they don't work in your browser, they won't work in the PURL server. Please make sure that they all resolve.

**Comment by shawntanzk** [comment]
oh yeah, it needs this to be merged first: https://github.com/obophenotype/cell-ontology/pull/1767
didn't know which order it should be in so was hoping to get approval and then merge them both together.

**Comment by jamesaoverton** [comment]
My understanding of that PR is that you're currently storing the release OWL files in git, but you're going to start attaching them to GitHub releases instead. So there's a synchronization problem. I see two options:

First, if you're not worried about downtime, then coordinate with me or @matentzn to merge this right after CL #1767.

Second, to avoid downtime and be more cautions, do this in three steps:

1. change the PURL config to point to the 2022-12-15 OWL files in git
2. merge CL #1767 and attach OWL files to the release
3. change the PURL config to point to the files attached to the GitHub release

**Comment by shawntanzk** [comment]
Sorry forgot to reply here, I think a bit of downtime is not a huge deal, I'll wait for https://github.com/obophenotype/cell-ontology/pull/1767 to be merged first, once that is done, I'll ping here and get this reviewed and merged :) thanks heaps

**Comment by shawntanzk** [comment]
@jamesaoverton I've changed CL release and the links should resolve now, would you mind helping me give a thumbs up if all is good? Thanks

**Comment by shawntanzk** [comment]
Thanks for doing this so fast @jamesaoverton! really appreciate it :)

**Comment by jamesaoverton** [comment]
Yes, it looks good. I'll merge now, so the PURLs will be updated in < 20 minutes. You can make further changes if needed.

**Pull request merged**
